<html><h1>Validation FAILED.</h1><p>Build Time: 27 minutes 36 seconds 359 ms</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x">7.1.x</a><br/>Branch GIT ID: <a href="https://github.com/liferay/com-liferay-commerce/commit/3b41d292a28fc64c65a96b3cd8e138166a7c45f7">3b41d292a28fc64c65a96b3cd8e138166a7c45f7</a></p><h6>Task Summary:</h6><ul><li>subrepository-compile-jdk8 -  :heavy_check_mark:</li><li>subrepository-semantic-versioning-jdk8 -  :heavy_check_mark:</li><li>subrepository-pmd-jdk8 -  :heavy_check_mark:</li><li>subrepository-source-format-jdk8 -  :x:<pre><code>     [exec] java.lang.Exception: Found 1 formatting issues:
     [exec] 1: Incorrect order 'setting': 'show-topbar' should come after 'show-sidebar': ./commerce-theme-minium/commerce-theme-minium/src/WEB-INF/liferay-look-and-feel.xml (SourceCheck:XMLLookAndFeelFileCheck)
     [exec] 
     [exec] 	at com.liferay.source.formatter.SourceFormatter.format(SourceFormatter.java:438)
     [exec] 	at com.liferay.source.formatter.SourceFormatter.main(SourceFormatter.java:266)
     [exec] 
     [exec] FAILURE: Build failed with an exception.
     [exec] 
     [exec] * What went wrong:
     [exec] Execution failed for task ':checkSourceFormatting'.
     [exec] &gt; Process 'command '/opt/java/jdk1.8.0_121-x64/bin/java'' finished with non-zero exit value 1
     [exec] 
     [exec] * Try:
     [exec] Run with --info or --debug option to get more log output. Run with --scan to get full insights.
     [exec] 
     [exec] * Exception is:
     [exec] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':checkSourceFormatting'.
     [exec] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:110)
     [exec] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:77)
     [exec] 	at org.gradle.api.internal.tasks.execution.OutputDirectoryCreatingTaskExecuter.execute(OutputDirectoryCreatingTaskExecuter.java:51)
     [exec] 	at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:59)
     [exec] 	at org.gradle.api.internal.tasks.execution.ResolveTaskOutputCachingStateExecuter.execute(ResolveTaskOutputCachingStateExecuter.java:54)
     [exec] 	at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:59)
     [exec] 	at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:101)
     [exec] 	at org.gradle.api.internal.tasks.execution.FinalizeInputFilePropertiesTaskExecuter.execute(FinalizeInputFilePropertiesTaskExecuter.java:44)
     [exec] 	at org.gradle.api.internal.tasks.execution.CleanupStaleOutputsExecuter.execute(CleanupStaleOutputsExecuter.java:91)
     [exec] 	at org.gradle.api.internal.tasks.execution.ResolveTaskArtifactStateTaskExecuter.execute(ResolveTaskArtifactStateTaskExecuter.java:62)</code></pre></li><li>central-requirements-jdk8/subrepository-unit-jdk8 -  :heavy_check_mark:</li></ul><h5>For full console, click <a href="https://test-1-14.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/139//consoleText">here</a>.</h5><hr/><div><h6>Test Results:</h6><div><div><p>18 Tests Passed.<br/>0 Tests Failed.</p></div></div><h5>For all test results, click <a href="https://test-1-14.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/139//testReport">here</a>.</h5></div></html>

--- a/commerce-initializer-util/src/main/java/com/liferay/commerce/initializer/util/PortletSettingsImporter.java
+++ b/commerce-initializer-util/src/main/java/com/liferay/commerce/initializer/util/PortletSettingsImporter.java
@@ -37,7 +37,6 @@ import com.liferay.portal.kernel.model.User;
 import com.liferay.portal.kernel.portlet.PortletIdCodec;
 import com.liferay.portal.kernel.portlet.PortletPreferencesFactoryUtil;
 import com.liferay.portal.kernel.service.ClassNameLocalService;
-import com.liferay.portal.kernel.service.CompanyLocalService;
 import com.liferay.portal.kernel.service.LayoutLocalService;
 import com.liferay.portal.kernel.service.ServiceContext;
 import com.liferay.portal.kernel.service.UserLocalService;
@@ -294,9 +293,6 @@ public class PortletSettingsImporter {
 
 	@Reference
 	private ClassNameLocalService _classNameLocalService;
-
-	@Reference
-	private CompanyLocalService _companyLocalService;
 
 	@Reference
 	private CPFileImporter _cpFileImporter;

--- a/commerce-theme-minium/commerce-theme-minium-site-initializer/src/main/resources/com/liferay/commerce/theme/minium/site/initializer/internal/dependencies/layouts.json
+++ b/commerce-theme-minium/commerce-theme-minium-site-initializer/src/main/resources/com/liferay/commerce/theme/minium/site/initializer/internal/dependencies/layouts.json
@@ -233,6 +233,18 @@
 	{
 		"hidden": true,
 		"layoutTemplateId": "2_columns_ii",
+		"lookAndFeel": {
+			"CUSTOMIZABLE_LAYOUT": false,
+			"column-1-customizable": false,
+			"column-2-customizable": false,
+			"layout-template-id": "2_columns_ii",
+			"layoutUpdateable": true,
+			"lfr-theme:regular:is-login-page": true,
+			"lfr-theme:regular:show-sidebar": false,
+			"lfr-theme:regular:show-topbar": false,
+			"sitemap-changefreq": "daily",
+			"sitemap-include": 1
+		},
 		"name": "Login",
 		"portlets": [
 			{

--- a/commerce-theme-minium/commerce-theme-minium/src/WEB-INF/liferay-look-and-feel.xml
+++ b/commerce-theme-minium/commerce-theme-minium/src/WEB-INF/liferay-look-and-feel.xml
@@ -9,8 +9,11 @@
 	<theme id="minium" name="Minium">
 		<template-extension>ftl</template-extension>
 		<settings>
+			<setting configurable="true" key="is-login-page" type="checkbox" value="false" />
 			<setting configurable="true" key="redirect-to-private-layouts" type="checkbox" value="false" />
+			<setting configurable="true" key="show-sidebar" type="checkbox" value="true" />
 			<setting configurable="true" key="show-top-menu" type="checkbox" value="false" />
+			<setting configurable="true" key="show-topbar" type="checkbox" value="true" />
 			<setting configurable="true" key="wide-layout" type="checkbox" value="false" />
 		</settings>
 		<layout-templates>

--- a/commerce-theme-minium/commerce-theme-minium/src/templates/init_custom.ftl
+++ b/commerce-theme-minium/commerce-theme-minium/src/templates/init_custom.ftl
@@ -1,6 +1,9 @@
 <#assign
+	is_login_page = getterUtil.getBoolean(themeDisplay.getThemeSetting("is-login-page"))
 	redirect_to_private_layouts = getterUtil.getBoolean(themeDisplay.getThemeSetting("redirect-to-private-layouts"))
 	show_top_menu = getterUtil.getBoolean(themeDisplay.getThemeSetting("show-top-menu"))
+	show_topbar = getterUtil.getBoolean(themeDisplay.getThemeSetting("show-topbar"))
+	show_sidebar = getterUtil.getBoolean(themeDisplay.getThemeSetting("show-sidebar"))
 	wide_layout = getterUtil.getBoolean(themeDisplay.getThemeSetting("wide-layout"))
 	userManagementUrl = commerceThemeMiniumHttpHelper.getAccountManagementPortletURL(request)
 	wishlistUrl = commerceWishListHttpHelper.getCommerceWishListPortletURL(request)

--- a/commerce-theme-minium/commerce-theme-minium/src/templates/portal_normal.ftl
+++ b/commerce-theme-minium/commerce-theme-minium/src/templates/portal_normal.ftl
@@ -16,11 +16,7 @@
 	</script>
 </#if>
 
-<#assign isMiniumLogin = false />
-
-<#if themeDisplay.isSignedIn() && themeDisplay.getLayoutSet().isPrivateLayout()>
-	<#assign isMiniumLogin = true />
-<#else>
+<#if is_login_page && !themeDisplay.isSignedIn()>
 	<#assign css_class = css_class + " minium-login" />
 </#if>
 
@@ -32,18 +28,20 @@
 	</div>
 
 	<main class="minium minium-frame" id="minium">
-		<#if isMiniumLogin>
+		<#if show_sidebar>
 			<div class="minium-frame__sidebar">
 				<#include "${full_templates_path}/sidebar.ftl" />
 			</div>
+		</#if>
 
+		<#if show_topbar>
 			<div class="minium-frame__topbar">
 				<#include "${full_templates_path}/topbar.ftl" />
 			</div>
-
-			<div class="minium-frame__content js-scroll-area">
-				<a name="minium-top"></a>
 		</#if>
+
+		<div class="minium-frame__content js-scroll-area">
+			<a name="minium-top"></a>
 
 			<div class="${minium_content_css_class}">
 				<#if selectable>
@@ -57,17 +55,16 @@
 					</@>
 				</#if>
 			</div>
-		<#if isMiniumLogin>
-			</div>
 
-			<#--  The toolbar is needed to create the shadow when scrolling  -->
+		</div>
 
-			<div class="minium-frame__toolbar"></div>
+		<#--  The toolbar is needed to create the shadow when scrolling  -->
 
-			<div class="minium-frame__overlay">
-				<@liferay_commerce_ui["search-results"] />
-			</div>
-		</#if>
+		<div class="minium-frame__toolbar"></div>
+
+		<div class="minium-frame__overlay">
+			<@liferay_commerce_ui["search-results"] />
+		</div>
 	</main>
 
 	<div class="liferay-bottom">

